examples/README.md

```diff
@@ -0,0 +1,141 @@
+# krun example
+
+This directory contains a simple experiment using krun.
+This is a good starting point for setting up your own krun configuration.
+
+The example here contains two benchmark programs (*nbody* and *dummy*),
+executed on two VMs (*cPython* and a standard *JVM* such as HotSpot).
+Each benchmark is run for 5 iterations on the same VM, then the VM is
+restarted and the benchmark is re-run for another 5 iterations.
+We say that the experiment runs 2 *executions* and 5 *iterations* of
+each benchmark.
+
+This configuration can be found in the file `examples/example.krun`.
+
+## Step 0: prepare the benchmarking machine
+
```

Comment:
Step 0? Step 1?!

Anyway, I think we should say something like "on Linux, you can optionally set up some kernel arguments to obtain more reliable results". On non-Linuxes, either there are no equivalent options, or there are different ones. We don't want to make it seem it's Linux only though.